src/credentials/sso/clientRegistration.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface ClientRegistration {
```

Comment:
ditto with above on the comment front, `ClientRegisteration` is probably a specific enough name

Reply:
Changed to SsoClientRegistration